packages/terra-clinical-label-value-view/src/terra-dev-site/doc/example/LabelValueViewTextTermAndDef.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import LabelValueView from 'terra-clinical-label-value-view';
+
+const LabelValueViewText = () => (
+  <div>
+    <dl>
```

Comment:
Not a huge deal if crunched on time but when childOfDescriptionList="true", can we just have one `<dl>` for all of the LabelValueViews rather than one for each? Same for the other examples as well.